File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.MoreEx_JS_Conditional_Stitiments-More Exercises/P02_SleepyTomCat.js
function sleepyTomCat(input) {
    let hollydays = Number(input[0]);
    let playTimeMins = (365 - hollydays) * 63 + hollydays * 127;

    if (playTimeMins > 30000) {
        console.log("Tom will run away");
        console.log(`${Math.floor((playTimeMins - 30000) / 60)} hours and ${(playTimeMins - 30000) % 60} minutes more for play`)
    } else {
        console.log("Tom sleeps well");
        console.log(`${Math.floor((30000 - playTimeMins) / 60)} hours and ${(30000 - playTimeMins) % 60} minutes less for play`)
    }
}

sleepyTomCat(["113"]);

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P02AdditionalProblems_ConditionalStatements/P02_SleepyTomCat.java
package P02AdditionalProblems_ConditionalStatements;

import java.util.Scanner;

public class P02_SleepyTomCat {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int holidays = Integer.parseInt(scanner.nextLine());

        int timePlay = 365 * 63 + holidays * 64;

        if (timePlay > 30000) {
            System.out.println("Tom will run away");
            System.out.printf("%d hours and %d minutes more for play",
                    (timePlay - 30000) / 60, (timePlay - 30000) % 60);
        } else {
            System.out.println("Tom sleeps well");
            System.out.printf("%d hours and %d minutes less for play",
                    (30000 - timePlay) / 60, (30000 - timePlay) % 60);
        }
    }
}
